src/components/userCard.tsx

```diff
@@ -0,0 +1,27 @@
+import projectsCardIcon from "../assets/icons/projectsCardIcon.svg";
+import { UserCardProps } from "@/interfaces/userInterface";
+
+export function UserCard(props: UserCardProps) {
+  const { data } = props;
+  const imageUrl = data.imageUrl ?? "";
```

Comment:
Is data.imageUrl always have a value? Because if you're creating a fallback in the variable to an empty string, it means that imageUrl might be undefined or null. In this case, you need to fix it in the interface to correctly tell Typescript what to expect here, and therefore replace for a data?.imageUrl with question mark

Reply:
I set like this because when I get the data from the graphql I can get empty, but I'm going to see if there is a alternative

Reply:
You don't need to change a lot in this particular line - just add question mark to turn it into data?.imageUrl. Also, you need to change the interface so it reflects correctly that it is an optional property.

Reply:
oh yeah, i changed to what you said it and it worked, i also change inside the Avatar props to work